testscript/testdata/kill.txt

```diff
@@ -0,0 +1,13 @@
+[!exec:sleep] skip
+
+# This test depends on sleep exiting with a non-success status when being
+# terminated by an interrupt (kill on Windows) signal.
+
+! exec sleep 10 &test_sleep&
+
+# Set a timeout. If the kill below fails, this sleep will still be running
+# before the test exits and so the test will fail when it completes.
+! exec sleep 5 &
+
+kill -KILL test_sleep
+wait test_sleep
```

Comment:
These negated tests are not currently possible since the `kill` command is not negatable. This is consistent with `wait`.

The positive tests for without name are possible.